biohansel/utils.py

```diff
@@ -170,30 +146,67 @@ def is_gzipped(p: str) -> bool:
     return bool(re.match(r'^.+\.gz$', p))
```

Comment:
just a small thing, but perhaps a more descriptive variable name for "p"?

Reply:
Changed in 47fa20fae518aee051462ab5c9116422d5bd47db